src/sensor/ping.cpp

```diff
@@ -487,6 +472,29 @@ void Ping::request(int id)
 
 void Ping::setLastPingConfiguration()
 {
+    if(!link()->isWritable()) {
```

Comment:
```suggestion
    static bool once = false;
    if (once) {
        return; // or *disconnect*
    }
    
    if(!link()->isWritable()) {
```

Reply:
This will be called on every single message rxed from the device.

Reply:
your 'once' did not encapsulate the entire code that needed to be run 'once'. We were trying to set device values from saved settings every time a message was rxed.